gssh_test.go

```diff
@@ -150,14 +177,127 @@ func envTest(t *testing.T, cli *Client) {
 	assert.Equal(t, "Hello, GSSH!\n", string(output))
 }
 
-func TestMain(m *testing.M) {
-	flag.StringVar(&addr, "addr", "", "The host of ssh")
-	flag.StringVar(&user, "user", "root", "The username of client")
-	flag.StringVar(&passwd, "pass", "", "The password of user")
-	flag.StringVar(&key, "ssh-key", "", "The location of private key")
+var _testPrivateKey = []byte(`
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
🛑 **Gitleaks** has detected a secret with rule-id `private-key` in commit ca02f9b597fcd0862dbc8f5955e37b4ffc2ef052.
If this secret is a _true_ positive, please rotate the secret ASAP.

If this secret is a _false_ positive, you can add the fingerprint below to your `.gitleaksignore` file and commit the change to this branch.

```
echo ca02f9b597fcd0862dbc8f5955e37b4ffc2ef052:gssh_test.go:private-key:181 >> .gitleaksignore
```

Reply:
removed